src/Metadata/Tests/Resource/Factory/ParameterResourceMetadataCollectionFactoryTest.php

```diff
@@ -41,6 +41,7 @@ public function testParameterFactory(): void
         $filterLocator->method('get')->willReturn(new class implements FilterInterface {
             public function getDescription(string $resourceClass): array
             {
+                // @phpstan-ignore-next-line return.type
```

Comment:
Seems like we're passing volontary a wrong type to check it's handle in the method.